apps/bbsync/tests/test_query.py

```diff
@@ -1182,3 +1186,37 @@ def test_generate_requires_doc_text(
             assert {"name": "requires_doc_text", "status": requires_doc_text} in flags
         else:
             assert flags == []
+
+    @pytest.mark.parametrize(
+        "field_state,flag_value,should_convert",
+        [
+            # flags to convert
+            (Flaw.FlawNistCvssValidation.REQUESTED, "?", True),
+            (Flaw.FlawNistCvssValidation.APPROVED, "+", True),
+            (Flaw.FlawNistCvssValidation.REJECTED, "-", True),
+            # flag to ignore
+            (Flaw.FlawNistCvssValidation.NOVALUE, None, False),
+        ],
+    )
+    def test_generate_nist_cvss_validation(
+        self, field_state, flag_value, should_convert
+    ):
+        """
+        Tests that nist_cvss_validation field is correctly converted into a flag.
+        Other flag-producing fields are set not to produce flags for ease of testing.
```

Comment:
Nitpick: If you want, you can use the code below to avoid this note:

        if should_convert:
            assert {"name": "nist_cvss_validation", "status": flag_value} in flags
        else:
            assert {"name": "nist_cvss_validation", "status": flag_value} not in flags